CMakeLists.txt

```diff
@@ -101,21 +103,12 @@ if(TTK_BUILD_PARAVIEW_PLUGINS)
   add_compile_definitions("TTK_BUILD_PARAVIEW_PLUGINS")
 
   # Install location
-  if(NOT "$ENV{PV_PLUGIN_PATH}" STREQUAL "")
-    set(TTK_INSTALL_PLUGIN_DIR
-      $ENV{PV_PLUGIN_PATH}
-      CACHE
-      PATH
-        "Directory where the ParaView plugin will be installed"
-      )
-  else()
-    set(TTK_INSTALL_PLUGIN_DIR
-      ${ParaView_DIR}/../../../bin/plugins/
```

Comment:
thanks for combining your new solution `"${CMAKE_INSTALL_BINDIR}/plugins"` with this old setup (as discussed in the meetup)

Reply:
Done!

Reply:
awesome!

Reply:
Actually, after some experimentation and considerations, I think keeping this old mechanism is a bad idea (for regular users/devs, not for the package users):
- it introduces an absolute path inside the configuration, which can cause side-effects, especially if you have several installations and build directories. You may overwrite the plugin of a previous installation if you forget to unset `PV_PLUGIN_PATH` before the configuration step.
- the `paraview_plugin_build` function call inside paraview/CMakeLists.txt only accepts relative paths (to `$ttk_prefix`). To support this mechanism, I hard-coded the `$ttk_prefix/bin/plugins` path there. An `install` call in the same file manages the installation in the user-provided `PV_PLUGIN_PATH`, resulting in a duplication of the plugin.
- `PV_PLUGIN_PATH`, like `PATH`, may contain a list of paths. For example, you can load all the ParaView plugins inside `$pv_prefix/lib/paraview-5.8/plugins` + TTK at launch this way.